src/atoms/fonts/fonts.tsx

```diff
@@ -20,8 +21,49 @@ export const FontsBase = ({ config }) => {
         crossorigin="anonymous"
         href={`https://fonts.googleapis.com/css?family=${secondary.url}&display=swap`}
       />
+      <link rel="preload" as="style" crossorigin="anonymous" href={customFonts?.primary1} />
+      <link rel="preload" as="style" crossorigin="anonymous" href={customFonts?.secondary1} />
+      <link rel="preload" as="style" crossorigin="anonymous" href={customFonts?.primary2} />
+      <link rel="preload" as="style" crossorigin="anonymous" href={customFonts?.secondary2} />
     </Helmet>
   );
 };
 
+/**
+ * Amp library supports google fonts as well as custom fonts. This Fonts component accepts primary and secondary google fonts from BOlD AMP  settings `settings > configure > AMP` and four different custom fonts from render props `customFonts`. These custom font urls are passed to the link tags that are preloaded. 
+ * In case customFonts is passed in the config, it is rendered along with the fonts mentioned in BOLD.
+ *
+ * Custom fonts can be hosted on allowed origins or can be added in the public folder of your app. 
+ * If added in public folder, mention the asset host and path in the url like this "https://fea.assettype.com/malibu/assets/ 
+  MalibuHeadlineSemiBold.woff2". 
+ * 
+ * The following origins are allowlisted and allowed for font serving via link tags as per official AMP documentation:
```

Comment:
Reframe the sentence **The following origins are allowlisted and allowed ...**

Reply:
That line is from the original docs.